image.yaml

```diff
@@ -23,6 +23,14 @@ envs:
 modules:
       repositories:
           - path: modules
+          - name: cct_module
+            git:
+                url: https://github.com/jboss-openshift/cct_module.git
+                ref: 0.37.1
+          - name: jboss-eap-7-image
+            git:
+                url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                ref: EAP_724_CR2
```

Comment:
Please add:
```
packages:
      install:
            - unzip
```
section below the existing `modules / install` section yet (this would install the `unzip` binary). Without it, image build fails with msg like:
```
2019-10-24 11:46:03,793 cekit        INFO     Docker: Step 7/13 : RUN bash -x /tmp/scripts/sso/install
2019-10-24 11:46:03,826 cekit        INFO     Docker: ---> Running in a4ea5cae9446
2019-10-24 11:46:03,828 cekit        INFO     Docker: {}
2019-10-24 11:46:03,828 cekit        INFO     Docker: 
2019-10-24 11:46:04,247 cekit        INFO     Docker: + set -e
2019-10-24 11:46:04,247 cekit        INFO     Docker: + SOURCES_DIR=/tmp/artifacts
2019-10-24 11:46:04,248 cekit        INFO     Docker: + unzip -o /tmp/artifacts/keycloak-server-overlay.zip -d
2019-10-24 11:46:04,249 cekit        INFO     Docker: /tmp/scripts/sso/install: line 7: unzip: command not found
2019-10-24 11:46:04,453 cekit        ERROR    You can look inside the failed image by running 'docker run --rm -ti 07ee0c2591a9
 bash'
2019-10-24 11:46:04,453 cekit        ERROR    ('Image build failed, see logs above.', CekitError("Image build failed: 'The command 'bash -x /tmp/scripts/sso/install' returned a non-zero code: 127'",))
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/cekit/cli.py", line 256, in run
    builder.build()
  File "/usr/lib/python2.7/site-packages/cekit/builders/docker_builder.py", line 98, in build
    raise CekitError(msg, ex)
CekitError: ('Image build failed, see logs above.', CekitError("Image build failed: 'The command 'bash -x /tmp/scripts/sso/install' returned a non-zero code: 127'",))
```

Reply:
Thanks! done